### PR TITLE
Feat/5575 async api documentation

### PR DESCRIPTION
**Pull request by marcambier**
**Issue**

https://github.com/gravitee-io/issues/issues/5575

**Description**

Add asyncapi documentation support

### CODE REVIEW

File: src/components/documentation/page/page-asyncapi.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<div ng-if="!$ctrl.pageContent">
+  <gv-icon shape="text:align-center" style="--gv-icon--s: 96px"></gv-icon>
```
Comment (gaetanmaisse) [inline src/components/documentation/page/page-asyncapi.html:20]: You can put this in a CSS class ;)

File: src/components/documentation/page/page-asyncapi.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <div>No content to display</div>
+</div>
+
+<div ng-if="$ctrl.pageContent">
+  <asyncapi-component schema="{{$ctrl.pageContent}}" cssImportPath="assets/style/asyncapi-component.css"> </asyncapi-component>
```
Comment (gaetanmaisse) [inline src/components/documentation/page/page-asyncapi.html:25]: div can be merged into asyncapi-component I think
```html
<asyncapi-component ng-if="$ctrl.pageContent" schema="{{$ctrl.pageContent}}" cssImportPath="assets/style/asyncapi-component.css"> </asyncapi-component>
```

**Review by gaetanmaisse — COMMENTED** [review]
Can you squash all the commits except the 1st one into a single one as everything is deeply coupled and needs to be there to work. 

Also, can you rebase this branch on master (instead of merging master into this one)? It tense to be the process advocated by Graviters to avoid merge commit and a get a simpler and linear git history. Feel free to ping me if you want to discuss that :)

**Comment by marcambier** [comment]
> Can you squash all the commits except the 1st one into a single one as everything is deeply coupled and needs to be there to work.
> 
> Also, can you rebase this branch on master (instead of merging master into this one)? It tense to be the process advocated by Graviters to avoid merge commit and a get a simpler and linear git history. Feel free to ping me if you want to discuss that :)

ok, done

File: src/components/documentation/page/page-asyncapi.html (hunk 2 above)
Comment (marcambier) [reply src/components/documentation/page/page-asyncapi.html]: done

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**